src/Uno.UI.Toolkit/Storage/StorageFileHelper.cs

```diff
@@ -18,9 +19,20 @@ public partial class StorageFileHelper
 	/// <returns>A task that will complete with a result of true if file exists, otherwise with a result of false.</returns>
 	public static async Task<bool> ExistsInPackage(string fileName) => await FileExistsInPackage(fileName);
 
+	/// <summary>
+	/// This asynchronous method retrieves the paths of files within a directory based on a specified filter for file extensions.
+	/// </summary>
+	/// <param name="extensionsFilter">An array of strings representing the file extensions to filter the files.If null, all files in the directory are considered.</param>
+	/// <returns>An array of strings containing the paths of the filtered files within the directory.</returns>
+	public static async Task<string[]> GetFilesInDirectoryAsync(string[] extensionsFilter)
```

Comment:
@rafael-rosa-knowcode By the look of the implementation, it means that you want all the files that `GetFileFromApplicationUriAsync` would provide. In this case, the method should be best called `GetFilesFromApplication`.